docs/technical_reference/unit_models/coag_floc_model.rst

```diff
@@ -0,0 +1,160 @@
+Coagulation-Flocculation
+========================
+
+Introduction
+------------
+
+Coagulation-Flocculation is a water treatment process  designed to remove total suspended
+solids (TSS, primarily as natural organic matter) from solution by converting those small,
+suspended particles into larger 'floccs' (or 'Sludge') that can then be separated by sedimentation
+and/or filtration (or other separation processes) later in the treatment train. This
+is accomplished in a 2-stage process:
+
+Stage 1) Coagulation (where chemical coagulants and other aids are rapidly mixed into solution)
+Stage 2) Flocculation (where the 'floccs' are formed through slow, gentle mixing/agitation)
+
+In this implementation of the model, the user MUST provide a measured final Turbidity (in NTU) made
+during a Jar Test for the given water source. This measurement is then used to estimate how much
+TSS would be removed during the Coagulation-Flocculation process. User's may also
+provide the specific chemical composition of additives used to achieve this final Turbidity, and
+can provide information on the level of salts in those additives. This information can be used
+to estimate an increase in total dissolved salts (TDS) that may occur due to the addition of
+those chemicals.
+
+This model also includes relationships for power usage in the rapid mixing and flocculation
+basins. User's will need to provide information for power usage such as retention times of
+each basin, mixing paddle sizes, number of mixers, etc.
+
+The main assumptions of the implemented model are as follows:
+
+1) Coagulation-Flocculation can be modeled together in a single combined unit (Figure 1)
+2) Model dimensionality is limited to 0D

```

Comment:
Does "Model dimensionality is limited to 0D" have a specific, technical meaning here? Or are you basically just saying "this is a simple for this unit"? If the latter, then I'd recommend rewording or removing this point, to avoid confusing the reader about what the meaning of "0D model" is.

Reply:
Thought about it a little more - does "0D" just mean that the model does not include a spatial component, in any dimensions?

Reply:
0D refers to the fact that the unit model is composed of a 0D control volume unit from IDAES, and that this unit does not have any dimensional variable, such as length or width for the control volume.